src/EnergyPlus/UnitarySystem.cc

```diff
@@ -4315,17 +4248,17 @@ namespace UnitarySystems {
         if (UtilityRoutines::SameString(loc_supFanPlace, "BlowThrough")) this->m_FanPlace = FanPlace::BlowThru;
         if (UtilityRoutines::SameString(loc_supFanPlace, "DrawThrough")) this->m_FanPlace = FanPlace::DrawThru;
         if (this->m_FanPlace == FanPlace::NotYetSet && this->m_FanExists) {
-            ShowSevereError(state, cCurrentModuleObject + " = " + thisObjectName);
-            ShowContinueError(state, "Illegal Fan Placement = " + loc_supFanPlace);
+            ShowSevereError(state, format("{} = {}", cCurrentModuleObject, thisObjectName));
+            ShowContinueError(state, format("Illegal Fan Placement = {}", loc_supFanPlace));
             errorsFound = true;
         }
 
         std::string loc_supFanOpMode = input_data.supply_air_fan_operating_mode_schedule_name;
```

Comment:
I know you didn't write this line, but `loc_supFanOpMode` is a deep copy of the string in `input_data`. Do you really need a copy here? Or is a reference (either `std::string const &` or `std::string_view`) sufficient? I would be the latter.

Reply:
I did write it and these loc_* were temporary and I began removing them in this branch.